app-store-connect/src/bundle_api.rs

```diff
@@ -0,0 +1,123 @@
+use crate::{AppStoreConnectClient, Result};
+use serde::{Deserialize, Serialize};
+
+const APPLE_CERTIFICATE_URL: &'static str = "https://api.appstoreconnect.apple.com/v1/bundleIds";
```

Comment:
Clippy is complaining about the `'static` annotation here. Might as well remove it.